core/src/main/java/org/jacorb/orb/giop/ClientConnectionManager.java

```diff
@@ -143,9 +143,9 @@ public synchronized ClientConnection getConnection(org.omg.ETF.Profile profile)
         }
```

Comment:
I think the creation (line 137) should be changed as well to match ?

Reply:
Made the log level change as per the review comment.